src/MonologCreator/Processor/RequestId.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace MonologCreator\Processor;
+
+/**
+ * Injects a per-request UUID into the log output.
+ */
+class RequestId
+{
+    /**
+     * @var string
+     */
+    protected $uuid;
+
+    public function __construct($uuid = null)
```

Comment:
why an optional parameter? I see this only in one unit test where you test that you only call the `_generateUUID()` method.